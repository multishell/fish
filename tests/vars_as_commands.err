fish: The expanded command was empty.
$EMPTY_VARIABLE
^
fish: The expanded command was empty.
"$EMPTY_VARIABLE"
^
