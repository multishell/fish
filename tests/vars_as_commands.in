# Test that using variables as command names work correctly.

$EMPTY_VARIABLE
"$EMPTY_VARIABLE"

set CMD1 echo basic command as variable
$CMD1

set CMD2 echo '(' not expanded again
$CMD2

# Test implicit cd
set CMD3 /usr/bin
$CMD3 && echo $PWD

exit 0
