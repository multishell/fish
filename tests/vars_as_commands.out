basic command as variable
( not expanded again
/usr/bin
